src/tests/ftest/osa/osa_online_drain.py

```diff
@@ -189,43 +116,39 @@ def run_online_drain_test(self, num_pool):
                 for thrd in threads:
                     self.log.info("Thread : %s", thrd)
                     thrd.start()
-                    time.sleep(5)
+                    time.sleep(1)
```

Comment:
hmm, hearing what Johann mentioned in one of out QA meetings, i think we should try to refrain from using sleep in our test code. Are you seeing any problem without sleep?

Reply:
Unfotunately, we need some timeouts here... I noticed ior jobs not started correctly... I am going to keep it right now.